lib/common/MongoDbTransactionStore.ts

```diff
@@ -158,11 +158,30 @@ export default class MongoDbTransactionStore implements ITransactionStore {
   }
 
   /**
-   * Gets a list of transactions that are in the desired transaction time
-   * @param transactionTime The transaction time to query for
+   * Gets a list of transactions between the bounds of transaction time.
+   * @param beginTransactionTime The first transaction time to query for (inclusive)
+   * @param endTransactionTime The transaction time to stop querying for (exclusive). Optional. If not provided, then only query for beginTransactionTime
    */
-  public async getTransactionsByTransactionTime (transactionTime: number): Promise<TransactionModel[]> {
-    const transactions = await this.transactionCollection!.find({ transactionTime: { $eq: Long.fromNumber(transactionTime) } }).toArray();
+  public async getTransactionsByTransactionTime (beginTransactionTime: number, endTransactionTime?: number): Promise<TransactionModel[]> {
```

Comment:
I suggest not having the ````endTransactionTime```` be optional. The function becomes much more simple.
````
start = Math.min(beginTransactionTime, endTransactionTime);
end = Math.max(beginTransactionTime, endTransactionTime);
...
````

Reply:
Using min max is a lot cleaner but it makes inclusive more arbitrary (the smaller value will always be inclusive and the larger will be exclusive), where as the current approach, start will always be inclusive and end will be exclusive, which makes it possible to have the behavior of something like "get from 10 (exclusive) to 100 (inclusive)"

Reply:
Gave it some thought and I think it's ok to use min max as currently we don't have a use case for looking back yet so the behavior doesn't really matter.